website_menu_by_user_status/models/website_menu.py

```diff
@@ -38,3 +36,26 @@ def _compute_visible(self):
             self.is_visible = self.user_not_logged
         else:
             self.is_visible = self.user_logged
+
+
+class WebsiteMenuVisibleForSpecificGroups(models.Model):
+    """Optionally display the menu item for a given list of user groups."""
+
+    _inherit = 'website.menu'
+
+    group_ids = fields.Many2many(
```

Comment:
you might want to have a look at this https://github.com/OCA/website/pull/394 as I proposed this improvement a long time ago :)

Reply:
@simahawk seems like you forgot some credentials in your module https://github.com/OCA/website/pull/394/files.

You barely copied website_menu_by_user_status, changed the name and added the feature.

I prefer augmenting the original one.

Reply:
are you kidding? read the full thread from the original PR :)

Reply:
@simahawk oups, right, did not read the first comment ;)

The purpose of website_menu_by_user_status is not about security. It is only to show or hide menu entries. But this was never mentioned in the module's README.

In my opinion, security should be applied on the content (blog post, web pages, etc) rather than menu entries.